test/e2e/save_test.go

```diff
@@ -116,6 +120,71 @@ var _ = Describe("Podman save", func() {
 		Expect(save).To(ExitWithError())
 	})
 
+	It("podman save remove signature", func() {
+		SkipIfRemote("podman-remote doesn't suppport tls-verify option")
```

Comment:
Podman remote now supports tls-verify.

Reply:
podman remove push --tls-verify works, but still see `unknown flag: --tls-verify` with podman pull,
in the test **int remote ubuntu-19 root host**
```
[+2546s] Running: /var/tmp/go/src/github.com/containers/podman/bin/podman-remote --remote --url unix:/run/podman/podman-1695fe65d29f17c37c0cd8754dbdd796e29036e2adea66ec67ab760001892f06.sock --remote --url unix:/run/podman/podman-1695fe65d29f17c37c0cd8754dbdd796e29036e2adea66ec67ab760001892f06.sock pull --tls-verify=false --signature-policy=sign/policy.json localhost:5000/alpine
[+2546s] Error: unknown flag: --tls-verify
```

Reply:
I opened a PR to fix this.

https://github.com/containers/podman/pull/8042